src/os/debug/fancierwindow.js

```diff
@@ -43,10 +43,6 @@ os.debug.FancierWindow.STYLE_RULES_ = goog.string.Const.from(
 os.debug.FancierWindow.prototype.writeInitialDocument = function() {
   os.debug.FancierWindow.superClass_.writeInitialDocument.call(this);
 
-  // close button should set the debug window to disabled
-  if (this.win) {
-    goog.events.listenOnce(this.win, goog.events.EventType.BEFOREUNLOAD, this.onBeforeUnload_, false, this);
```

Comment:
I think the intent of this handler was to close the log window when the application was closed/refreshed. It should be listening to `window` (the application window) instead of `this.win` (the log window). Please make that change, and also remove the `unlisten` call below because `listenOnce` will remove the handler when the callback fires.

Reply:
Will do. Thanks for the feed back.